woocommerce/compatibility/class-sv-wc-order-compatibility.php

```diff
@@ -0,0 +1,298 @@
+<?php
+/**
+ * WooCommerce Plugin Framework
+ *
+ * This source file is subject to the GNU General Public License v3.0
+ * that is bundled with this package in the file license.txt.
+ * It is also available through the world-wide-web at this URL:
+ * http://www.gnu.org/licenses/gpl-3.0.html
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@skyverge.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade the plugin to newer
+ * versions in the future. If you wish to customize the plugin for your
+ * needs please refer to http://www.skyverge.com
+ *
+ * @package   SkyVerge/WooCommerce/Compatibility
+ * @author    SkyVerge
+ * @copyright Copyright (c) 2013-2017, SkyVerge, Inc.
+ * @license   http://www.gnu.org/licenses/gpl-3.0.html GNU General Public License v3.0
+ */
+
+defined( 'ABSPATH' ) or exit;
+
+if ( ! class_exists( 'SV_WC_Order_Compatibility' ) ) :
+
+/**
+ * WooCommerce order compatibility class.
+ *
+ * @since 4.6.0-dev
+ */
+class SV_WC_Order_Compatibility extends SV_WC_Data_Compatibility {
+
+
```

Comment:
should indicate here what order this array is in, i'm assuming `new property => old property`?